### ISSUE TITLE
0..200% in Klipper Menu Control

### ISSUE THREAD

**Issue by SergeyKrutyko** [KevinOConnor/klipper#644]
i get solution to change print speed from menu:
...
[menu control_prtspeed]
type: input
name: "Speed: {0:4.0%}"
parameter: gcode.speed_factor
transform:
	(0,1,0,100)
input_min: 0
input_max: 1
input_step: 0.1
gcode: M220 S{1:d}
...
now i set speed from 0 to 100%.... but i need from 0...200%
help me?

**Comment by mcmatrix** [comment]
You can try this
```ini
[menu control_prtspeed]
type: input
name: "Speed: {0:4.0%}"
parameter: gcode.speed_factor
transform:
    map(0,2,0,200)
input_min: 0
input_max: 2
input_step: 0.01
gcode: M220 S{1:d}
```

**Comment by KevinOConnor** [comment]
This looks like it was merged as part of commit 7b6a44e8.
